.github/workflows/test.yml

```diff
@@ -47,9 +46,8 @@ jobs:
           pip3 install -r requirements_doc.txt
           mkdir -p build
           cd build
-          cmake -DBUILD_TESTS_EXAMPLES=ON ..
```

Comment:
You probably still want this so it actually tests macos. No binary is packaged here so it fine if you want to avoid updating the license

Reply:
Ok.